config.schema.yml

```diff
@@ -413,6 +413,10 @@ properties:
                         matrixClients:
                             type: "object"
                             properties:
+                                localPartCharacterMapping:
+                                    type: "integer"
+                                    minimum: 0
+                                    maximum: 1
```

Comment:
Is there a way to set a default value here so it doesn't break existing configs? Or does that happens automatically